mrmustard/physics/gaussian.py

```diff
@@ -574,34 +573,35 @@ def general_dyne(
             oucome probability [units of `\hbar**N`],
             post-measurement covariace [units of `2\hbar`]
             post-measurement means vector [units of `\sqrt{\hbar}`].
+            pdf (Tensorflow probability distribution)
```

Comment:
If the numpy backend is used, it is not a tensorflow distribution

Reply:
Yes it is. Here we are using TF Probability, not TF.

Reply:
No, I don't think so. Look at the MultivariateNormalTriL function in numpy_backend.py. It does not return a TF object. Let's touch base on this later, before you resolve this convo